components/base/sensorcontrolled/spin.go

```diff
@@ -137,17 +154,6 @@ func calcAngVel(angErr, degsPerSec, slowDownAng float64) float64 {
 	return angVel
 }
 
-// getAngError computes the current distance the spin has moved and returns how much further the base must move to reach the goal.
-func getAngError(currYaw, prevAngle, prevMovedAng, desiredAngle float64) (float64, float64) {
-	// use initial angle to get the current angle the spin has moved
-	angMoved := getMovedAng(prevAngle, currYaw, prevMovedAng)
-
-	// compute the error
-	errAng := (desiredAngle - angMoved)
-
-	return errAng, angMoved
-}
-
```

Comment:
ended up being better to break up this function to make the shared components easier to share